jest.config.base.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    bail: true,
+    verbose: true,
+    testRegex: "(/src(/?[^/]*){0,5}/__tests__/).*Spec\\.jsx?$",
+    modulePaths: [
+		"<rootDir>/"
+    ],
+    testPathIgnorePatterns: ["__snapshots__", "/node_modules/"]

```

Comment:
There is no need for this key to be configured, its default value is `node_modules` and since snapshot folder contains only .snap files, there will be no match for tests.

Reply:
key removed